/**:
  ros__parameters:
    cam_base_topic: camera/image_raw
    output_frame: ''

    marker_dict: ARUCO_ORIGINAL

    image_sub_qos:
      reliability: 2 # 0 - system default, 1 - reliable, 2 - best effort
      durability: 2 # 0 - system default, 1 - transient local, 2 - volatile
      depth: 1

    publish_tf: true
    marker_size: 0.100

    # Dynamically reconfigurable Detector parameters
    # https://docs.opencv.org/4.2.0/d5/dae/tutorial_aruco_detection.html
    aruco:
      adaptiveThreshWinSizeMin: 3
      adaptiveThreshWinSizeMax: 23
      adaptiveThreshWinSizeStep: 10
      adaptiveThreshConstant: 7.0
      minMarkerPerimeterRate: 0.03
      maxMarkerPerimeterRate: 4.0
      polygonalApproxAccuracyRate: 0.03
      minCornerDistanceRate: 0.05
      minDistanceToBorder: 3
      minMarkerDistanceRate: 0.05
      markerBorderBits: 1
      perspectiveRemovePixelPerCell: 4
      perspectiveRemoveIgnoredMarginPerCell: 0.13
      maxErroneousBitsInBorderRate: 0.35
      minOtsuStdDev: 5.0
      errorCorrectionRate: 0.6
      cornerRefinementMethod: 2 # 0 - None, 1 - Subpix, 2 - Contour
      cornerRefinementWinSize: 5
      cornerRefinementMaxIterations: 30
      cornerRefinementMinAccuracy: 0.1
